config/jobs/kubernetes/kubeadm/kubeadm-presubmits.yaml

```diff
@@ -0,0 +1,12 @@
+# k8s.io/kubeadm presubmits
+presubmits:
+  kubernetes/kubeadm:
+  - name: pull-kubeadm-kinder-verify
```

Comment:
Do you want this on a testgrid dashboard? Edit testgrid/config.yaml if so!

Reply:
nope, not for now.